src/components/tables/RowActions/Shared/Confirmation.tsx

```diff
@@ -1,29 +1,48 @@
 import { List, ListItem, Typography } from '@mui/material';
 import { ReactNode } from 'react';
+import { SelectTableStoreNames } from 'stores/names';
+import NestedListItem, { SettingMetadata } from './NestedListItem';
 
-interface RowActionConfirmationprops {
+interface RowActionConfirmationProps {
     message: ReactNode;
     selected: any; //SelectableTableStore['selected'];
+    selectableTableStoreName?:
+        | SelectTableStoreNames.CAPTURE
+        | SelectTableStoreNames.COLLECTION
+        | SelectTableStoreNames.COLLECTION_SELECTOR
+        | SelectTableStoreNames.MATERIALIZATION;
+    settings?: SettingMetadata[];
 }
 
 function RowActionConfirmation({
     message,
     selected,
-}: RowActionConfirmationprops) {
+    selectableTableStoreName,
+    settings,
+}: RowActionConfirmationProps) {
+    const renderListItems =
+        settings && settings.length > 0 && selectableTableStoreName
+            ? (value: string) => (
+                  <NestedListItem
+                      key={`confirmation-selected-items-${value}`}
+                      catalogName={value}
+                      selectableTableStoreName={selectableTableStoreName}
+                      settings={settings}
+                  />
+              )
+            : (value: string) => (
+                  <ListItem key={`confirmation-selected-items-${value}`}>
+                      <Typography component="span" sx={{ fontWeight: 500 }}>
+                          {value}
+                      </Typography>
+                  </ListItem>
+              );
+
     return (
         <>
             {message}
-            <List>
-                {selected.map((value: string) => {
-                    return (
-                        <ListItem key={`confirmation-selected-items-${value}`}>
-                            <Typography variant="h6" component="span">
-                                {value}
-                            </Typography>
-                        </ListItem>
-                    );
-                })}
-            </List>
+
+            <List>{selected.map(renderListItems)}</List>
```

Comment:
Since the `NestedListItem` is rendering divs - we should mark this `List` with a `div` component and then update the `ListItem` up above.